pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -1023,6 +1023,7 @@ func generateContainerID(r *rand.Rand) string {
 	return strconv.FormatInt(int64(id), 10)
 }
 
+//nolint:unused // uniqueSandboxID is used in rego tests
```

Comment:
Is it really that this is called in uniqueSandboxID which is used in the tests?

Reply:
yeah, the only place generateSandboxID is called is from uniqueSandboxID, and that is only used in the rego tests
which the linter now doesn't look at, cause that excluded a couple files that have a !rego constraint